.github/workflows/ntbs-test.yml

```diff
@@ -1,9 +1,7 @@
 name: run tests
 
 on:
-  push:
-    branches-ignore:
-      - master
+  push
```

Comment:
Is this definitely still valid without the colon?

Reply:
For consistency, it might be better putting it back in anyway

Reply:
I think we need to not have the colon; I believe that the colon signifies that it is conditional, and the lack of one means 'all pushes': https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-a-single-event

I don't know if it's still valid with the colon, but seems best to go with docs